apps/coscrad-frontend/src/components/home/home.tsx

```diff
@@ -1,16 +1,28 @@
 import { useAuth0 } from '@auth0/auth0-react';
+import { Typography } from '@mui/material';
 import { useContext, useEffect } from 'react';
 import { useAppDispatch } from '../../app/hooks';
 import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
 import { userLoginSucceeded } from '../../store/slices/auth';
+import { ImageInContent } from '../image-in-content/image-in-content';
 
 export const Home = (): JSX.Element => {
+    /**
+     * TODO: Move data (ConfigurableContentContext, auth, and dispatch) out of presenter
+     */
     const { siteDescription, siteHomeImageUrl } = useContext(ConfigurableContentContext);
 
     const { isAuthenticated, getAccessTokenSilently, user } = useAuth0();
 
     const dispatch = useAppDispatch();
 
+    const image = {
```

Comment:
Add a TODO to consider using a `Photograph` detail presenter for this. Eventually we want to specify photographs from the db for logos.

Reply:
Done